### PR TITLE
Create net.mancubus.SLADE manifest

### PR DESCRIPTION
**Pull request by Alexander-Wilms**

### CODE REVIEW

**Comment by nedrichards** [comment]
testbot, build net.mancubus.SLADE
bot, build net.mancubus.SLADE

**Comment by nedrichards** [comment]
@Alexander-Wilms is upstream interested in co-maintaining? I see there was some good progress around improving the appdata.

**Comment by Alexander-Wilms** [comment]
The auther did seem to be interested in Flatpaks: https://github.com/sirjuddington/SLADE/issues/851

@sirjuddington Would you like to have push-access to the Flathub repository?

File: net.mancubus.SLADE.yaml [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+- --persist=.config/gzdoom
+- --filesystem=home
+cleanup:
+- /app/include
+- /app/lib/*.a
+- /app/lib/*.la
+- /app/lib/pkgconfig
+modules:
+
+- name: webkitgtk
```
Comment (barthalion) [inline net.mancubus.SLADE.yaml]: Isn't it already part of GNOME platform?

Comment (Alexander-Wilms) [reply net.mancubus.SLADE.yaml]: True, I've switched to the Gnome SDK.

**Comment by nedrichards** [comment]
testbot, build net.mancubus.SLADE
bot, build net.mancubus.SLADE

**Comment by nedrichards** [comment]
testbot, build net.mancubus.SLADE

**Comment by nedrichards** [comment]
bot, build net.mancubus.SLADE

**Comment by nedrichards** [comment]
A bunch of dconf errors, I assume you need to add the usual permissions:

> (slade:3): dconf-WARNING **: 20:51:09.485: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

**Comment by Alexander-Wilms** [comment]
Added the permissions, haven't tested it yet.

**Comment by Alexander-Wilms** [comment]
bot, build net.mancubus.SLADE

**Comment by Alexander-Wilms** [comment]
The warnings are gone.

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/net.mancubus.SLADE

**Comment by nedrichards** [comment]
Let me know if there's anyone else we should add to the repo. Don't forget there's new features at flathub, and in particular there's an inbuild delay to releases to allow for extra testing, although you can release whenever you want https://blogs.gnome.org/alexl/2019/02/19/changes-in-flathub-land/

**Pull request closed without merging**